src/navigation/wallet-connect/components/Connections.tsx

```diff
@@ -47,7 +47,8 @@ export default ({
   wallet: Wallet;
   connectors: IWCConnector[];
 }) => {
-  const {walletName, network, chain} = wallet.credentials;
+  const {walletName, credentials} = wallet;
```

Comment:
it should not be necessary to extract `credentials` with the recent changes in the wallet transform function.